lib/idl/idl_checkers.es6.js

```diff
@@ -65,5 +65,14 @@ module.exports = [
     logger.info('Checking top-level member names');
 
     eachKey(logger, left.members || [], right.members || [], 'name');
+  },
+  function rightHavingNoInterfaceObject(logger, left, right) {
+    const chk = check.bind(this, logger);
+    chk(
+      !(left.attrs && left.attrs.some(attr => attr.name === 'NoInterfaceObject')) && 
+      (right.attrs && right.attrs.some(attr => attr.name === 'NoInterfaceObject')),
+      null,
+      `${left.name}: NoInterfaceObject in Blink, but not in spec`
```

Comment:
Please use "left" and "right". The tool is intended to work with other data sets too!

E.g., `${left.name}: NoInterfaceObject in right, but not in left`